tests/bluetooth/bsim_bt/bsim_test_mesh/tests_scripts/heartbeat/sub_cb_api_unicast.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# Copyright 2021 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+source $(dirname "${BASH_SOURCE[0]}")/../../_mesh_test.sh
+
+# Run unicast heartbeat subscription to a device
+RunTest mesh_heartbeat_sub_cb_api_unicast \
+	heartbeat_publish_unicast \
+	heartbeat_publish_unicast \
```

Comment:
Why is it run twice?

Reply:
It initializes two devices who is trying to publish HB to the third subscription device. The subscription device should only receive hearbeats from the device it is subscribing to.